SYS_ATL/new_eff.py

```diff
@@ -1156,8 +1156,8 @@ class ES(Enum):
 class SchedulingError(Exception):
     def __init__(self, message, **kwargs):
         ops = self._get_scheduling_ops()
-        # TODO: include outer ops in message
```

Comment:
LGTM if this change in new_eff.py don't cause merge conflict with @gilbo 's branch

Reply:
I can revert that for now. It was just to get a better error message while I was debugging.